src/pipelines/inline.rs

```diff
@@ -84,7 +85,7 @@ impl FromStr for ContentType {
     type Err = anyhow::Error;
 
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        match s {
+        match dbg!(s) {
```

Comment:
Let's revert this. As is, it will cause a random line to be printed with the contents ... I'd prefer for that not to be the case.

Reply:
Well, that's embarrassing. I overlooked that one, sorry.